hudi-common/src/test/java/org/apache/hudi/common/testutils/reader/HoodieTestReaderContext.java

```diff
@@ -78,7 +80,9 @@ public ClosableIterator<IndexedRecord> getFileRecordIterator(
       Schema requiredSchema,
       StorageConfiguration<?> conf
   ) throws IOException {
-    HoodieAvroParquetReader reader = new HoodieAvroParquetReader(conf, new StoragePath(filePath.toUri()));
+    HoodieAvroFileReader reader = (HoodieAvroFileReader) HoodieFileReaderFactory
+        .getReaderFactory(HoodieRecord.HoodieRecordType.AVRO).getFileReader(null,
```

Comment:
Instead of `null`, should the empty `HoodieConfig` be passed in here, to avoid NPE if the underlying implementation is changed to leverage the Hudi configs?